src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php

```diff
@@ -103,16 +104,29 @@ protected function processItem($WpObject)
         }
 
         $this->debug('Added guest information', $callData);
+
+        $ShopModule = $this->storekeeper_api->getModule('ShopModule');
+
         /*
          * Order products
          */
         if (!$isUpdate) {
             $callData['order_items'] = $this->getOrderItems($WpObject);
             $this->debug('Added order_items information', $callData);
         } else {
-            $callData['order_items__do_not_change'] = true;
-            $callData['order_items__remove'] = null;
-            $this->debug('Update, ignored order_items', $callData);
+            $storekeeperId = $this->get_storekeeper_id();
+            $storekeeperOrder = $ShopModule->getOrder($storekeeperId, null);
+
+            $hasDiscrepancy = $this->checkOrderDiscrepancy($WpObject, $storekeeperOrder);
```

Comment:
```suggestion
            $hasDifferences = $this->checkOrderItemDifference($WpObject, $storekeeperOrder);
```
discrepancy is illogical :P

Reply:
Changed `discrepancy` word to `difference`.